src/Diagnostics.DataProviders/DataProviders/Base/SupportObserverDataProviderBase.cs

```diff
@@ -25,45 +26,61 @@ public SupportObserverDataProviderBase(OperationDataCache cache, SupportObserver
             FillObserverRouteTemplates();
         }
 
-        public async Task<dynamic> GetResource(string observerUrl)
+        public Task<dynamic> GetResource(string resourceUrl)
```

Comment:
will changing the name of the parameter in the method signature mean that all the detectors that use this (not sure how many) need to be recompiled?

Reply:
Nope